connect/api.go

```diff
@@ -146,6 +150,24 @@ func deregisterSystem() error {
 	return err
 }
 
+func synchronizeProducts(products []Product) ([]Product, error) {
```

Comment:
kinda long name, maybe syncProducts() or synchronize() like the Ruby version

Reply:
`synchronize()` looks a bit more generic to me. `syncProducts()` is fine.

Reply:
done